tasks/rsync-deploy.yml

```diff
@@ -1,19 +1,24 @@
 ---
+
+# Migration check from symlink deployment
+- stat:
+    path: "{{ ansistrano_deploy_to }}/{{ ansistrano_current_dir }}"
+  register: stat_current_dir
+
+- name: ANSISTRANO | Remove current folder if it's a symlink
+  file:
+    state: absent
+    path: "{{ ansistrano_deploy_to }}/{{ ansistrano_current_dir }}"
+  when: stat_current_dir.stat.islnk is defined and stat_current_dir.stat.islnk
+
 # Perform rsync deployment
 - name: ANSISTRANO | Ensure current folder is a directory
   file:
     state: directory
     path: "{{ ansistrano_deploy_to }}/{{ ansistrano_current_dir }}"
 
 - name: ANSISTRANO | Sync release to new current path
-  synchronize:
-    src: "{{ ansistrano_release_path.stdout }}/"
-    dest: "{{ ansistrano_deploy_to }}/{{ ansistrano_current_dir }}/"
-    set_remote_user: "{{ ansistrano_rsync_set_remote_user }}"
-    recursive: yes
-    delete: yes
-    archive: yes
-  delegate_to: "{{ inventory_hostname }}"
+  command: rsync -a -F --delete-after "{{ ansistrano_release_path.stdout }}/" "{{ ansistrano_deploy_to }}/{{ ansistrano_current_dir }}/"
```

Comment:
Wow, does the local rsync use known_hosts when using synchronize? That's bonkers!

Alright, will merge this tonight it as it does not affect the "main" Ansistrano path and you seem to need this :)

How does rollback end up in this scenario? I am not sure it will work as it relies on symlinks, have you tested it? Anyway, if it is broken it was broken since we merged your first iteration